.ci/install_clearcontainers_kernel.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repo_owner="clearcontainers"
+repo_name="linux"
+cc_linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
+
+set -e
+function usage() {
+    cat << EOT
+Usage: $0 <version>
+Install the containers clear kernel image <version> from clearcontainers/linux.
+
+version: Use latest to pull latest kernel or a version from ${cc_linux_releases_url}
+EOT
+
+exit 1
+}
+
+function download_kernel() {
+	local version=$1
+	local github_base="https://api.github.com/repos"
+	local latest_release_info_url="${github_base}/${repo_owner}/${repo_name}/releases/latest"
+	[ -n "${version}" ] || die "version not provided"
+	if [ "${version}" == "latest" ]; then
+		release_json="$(curl -s ${latest_release_info_url})"
+	parse_py='
```

Comment:
Nit: This line should be indented to line up with `release_json=` I think?

Reply:
I dont know an easy way to parse a json, I could use `jq` I dont know how common to be available for different distros. Because is a little python  if indent then the python interpreter will comply. I could do it in one line but then will be more difficult to read.

Reply:
Oh, sorry - I just meant the `parse_py=` line, but I agree that the python code below needs to stay "as-is". As I say, it's a "nit", so isn't a blocker. I agree - `jq` is great but won't necessarily be available so I think your Python solution is a good pragmatic one.